csharp/autobuilder/Semmle.Autobuild.Cpp.Tests/BuildScripts.cs

```diff
@@ -200,9 +200,9 @@ public void DownloadFile(string address, string fileName)
 
     internal class TestDiagnosticWriter : IDiagnosticsWriter
     {
-        public IList<DiagnosticMessage> Diagnostics { get; } = new List<DiagnosticMessage>();
+        public IList<Semmle.Util.DiagnosticMessage> Diagnostics { get; } = new List<Semmle.Util.DiagnosticMessage>();
 
-        public void AddEntry(DiagnosticMessage message) => this.Diagnostics.Add(message);
+        public void AddEntry(Semmle.Util.DiagnosticMessage message) => this.Diagnostics.Add(message);
```

Comment:
Are these changes needed?

Reply:
Yes - I think this is related to the strict dependencies problem I was discussing above with Paolo, or with the `xunit` upgrade. At this point, there's a `DiagnosticMessage` class visible from both `xunit` and `Semmle.Util`, so we have to disambiguate them.